src/lib/errors.js

```diff
@@ -24,9 +24,19 @@
 
 'use strict'
 
+const errorTitle = {
```

Comment:
I would rather rename this to "errorType" as this is not a `title`.

Reply:
Changed. Thanks for the review.